scripts/utils/namespaces-to-namespace-prefixes-file-generator.py

```diff
@@ -0,0 +1,127 @@
+#  #%L
+#  Alfresco HX Insight Connector
+#  %%
+#  Copyright (C) 2024 Alfresco Software Limited
+#  %%
+#  This file is part of the Alfresco software.
+#  If the software was purchased under a paid Alfresco license, the terms of
+#  the paid license agreement will prevail.  Otherwise, the software is
+#  provided under the following open source license terms:
+#
+#  Alfresco is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  Alfresco is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+#  #L%
+
+import sys
+import base64
+import requests
+import json
+
+PAGE_SIZE = 100
+ENDPOINT = "/alfresco/api/-default-/public/alfresco/versions/1/types"
+
+
+def is_input_valid():
+    return len(sys.argv) == 4 and not (None in sys.argv)
+
+
+def should_print_help():
+    return not is_input_valid() or sys.argv[1] == "help" or sys.argv[1] == "-h" or sys.argv[1] == "--help"
```

Comment:
I would use `argparse` for this. Even if it feels a little overkill then it makes the code simpler.

Reply:
meh I wanted to keep it super-simple and do not add this lib - idk if it'll add any readability. I cleaned up console args part - can you take a look one more time?

Reply:
Ok - let's look at this separately then. There's no fault in this PR anyway.